dapps/digital-twin-detail/src/components/container/DataSetForm.ts

```diff
@@ -152,6 +153,7 @@ export default class DataSetFormComponent extends mixins(EvanComponent) {
       }
       case 'object': {
         control.uiSpecs.type = 'json';
+        control.value = control.value || {};
```

Comment:
```suggestion
        control.value = control.value ?? {};
```
Consider using the nullish coalescing operator moving forward to declare default fallbacks :) It's a cool new feature.